android/automl/app/src/main/java/com/google/mlkit/vision/automl/demo/automl/AutoMLImageLabelerProcessor.java

```diff
@@ -69,7 +69,7 @@ public AutoMLImageLabelerProcessor(Context context, Mode mode) {
                         .addOnFailureListener(ignored ->
                                 Toast.makeText(
                                         context,
-                                        "Model download failed for AutoMLImageLabelerImpl,"
+                                        "Model download failed for '"+remoteModelName+"',"
```

Comment:
Use space to separate "+" and variable.

Reply:
Good catch 👍 I solved it

But going deeper into this issue, I'm wondering why a `./gradlew check` doesn't find it ? 
I test in my fork this with a CI action https://github.com/hannesa2/mlkit/pull/1/checks?check_run_id=996736299 and there is no ktLintCheck !

So I found out two things:
* Using your ktLint library is not a good idea, because it has a known vulnerability https://snyk.io/vuln/maven:com.github.shyiko%3Aktlint
* Your ktLint setup is not working at all with `./gradlew check`

Reply:
I dropped the `ktLint` library update and add more findings